source/get-started/cloud-install/qingcloud.rst

```diff
@@ -25,8 +25,8 @@ This tutorial assumes that you have completed the following configuration steps:
   `website <https://www.qingcloud.com>`_.
 
 
-Select and start |CL| virtual host QingCloud console
-*****************************************************
+Select and start |CL| virtual host with QingCloud console
```

Comment:
Ln 20: `--what is this for?...` I think this is a leftover comment that should be deleted.